tools/tuhi-kete-sandboxed.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+
+import sys
+import multiprocessing
+
+
+def start_or_not_tuhi(queue):
```

Comment:
that's a bit too french (I think). pls rename to 'maybe_start_tuhi` or something